{Icemoon, Ruin} The Crashed Holemaker

unit0016 opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
Adds a brand-new - well, [new to shiptest](https://cohost.org/bluwu016/post/885120-a-compilation-of-my) - ruin to the Icemoon roster, focused on the service department.

It's flavored around being an *incredibly* old NT Spaceworks vessel that's been carved in half and crashed - what's present only being the fore of the ship. Being mainly service-focused, it's loot is pretty dry ~~as is it's sole threat.~~
If more current-day mappers/balance-heads have any words about how to fluff out either of those pools a bit more with the screenshots below, lemme know. I'll listen well.

(Notarized loot summary removed as updating it was a pain in the ass, lmao.)

It strikes me as leaning on the underwhelming side from looking at the other ruins present here but we'll. See? I suppose? It's good practice for me in the whole, "making something I have memorized and that looks good normally look sicker ruined".

<details><summary>Pictures (All but SDMM Outdated)</summary>
<p>

Ignore that there's no rust, the firelocks are open here, and some stuff's knocked around, I was testing it prior to me tacking the rust on and took pics after running around it in-person.

![image](https://github.com/shiptest-ss13/Shiptest/assets/50649185/51a3cc54-1727-4241-9592-639f892621bf)

![image](https://github.com/shiptest-ss13/Shiptest/assets/50649185/580e39fe-bbf9-481f-aff8-cc25f860638d)

StrongDMM View:

![2023-11-09 15 02 20](https://github.com/shiptest-ss13/Shiptest/assets/50649185/5b94af63-d2d8-42bd-a823-0d300d66191f)

</p>
</details> 
<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game
This isn't what I intended to do when I was like, "oh yeah, I have a goofy ahh downstream out of boredom, ya'll want some of our better ships" but w/e here it is anyways. Ya'll need ruins. I made (another) ruin.
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog

:cl:
add: A new icemoon ruin has been added, should you be in need of service department goodies.
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

unit0016 committed:
Crashed Holemaker, Scribbles Of A Bored Mapper

unit0016 committed:
rusteze

spockye reviewed (commented):
we could probably use some less loot-intensive ruins 
Not seeing any issues, I'll give it a TM Soonish!

Latentish reviewed (changes requested):
Make a ruin catalogue entry.

unit0016 committed:
Map Catalogue, Blacklist Entry

unit0016 commented:
Addressed

Latentish commented on config/iceruinblacklist.txt:
```diff
@@ -16,3 +16,4 @@
 #_maps/RandomRuins/IceRuins/icemoon_underground_brazillianlab.dmm
 #_maps/RandomRuins/IceRuins/icemoon_underground_burnies_lair.dmm
 #_maps/RandomRuins/IceRuins/icemoon_underground_wendigo_cave.dmm
+#_maps/RandomRuins/IceRuins/icemoon_crashed_holemaker.dmm
```
This is a deprecated file and is not being maintained, nor should be altered.

unit0016 committed:
Revert iceruinblacklist changes

Signed-off-by: BluBerry016 <50649185+unit0016@users.noreply.github.com>

Latentish commented:
This seems like a good stash of loot for something with no real threats.. Also I think you could benefit from adding some exterior padding, some of the environment around it, so by default it doesn't spawn out in the open.

unit0016 committed:
Merge branch 'master' of https://github.com/shiptest-ss13/Shiptest into tonight's-sosiest-loser

unit0016 commented:
> This seems like a good stash of loot for something with no real threats.. 

I'll update this, but honestly, I *really* don't believe that it's the case.
Even ignoring that other pre-existing ruins are thousands times more egregious (As examples I can name off the top of my head from playing, see `icemoon_surface_engioutpost.dmm`'s free lathe setup and full engineering kit for three people and `jungle_surface_roommates.dmm` for two *40 force highlander claymores*), the two big ticket items are.. a bar setup (two bar dispensers, a reagent grinder, a maintenance booze-o-mat, a condimaster) - and a service protolathe\*, which is more restricted than even the autolathe, save a *few* items that can't be moved over to an autolathe via a design disk and RND.

Everything else is either redundant or otherwise incredibly minor.

\* NOT a techfab, to be clear. This can't print circuit boards. Yes, they look IDENTICAL. It annoys the hell out of me too!! I tried to fix this on /tg/ but a conflict happened and I got lazy.

> Also I think you could benefit from adding some exterior padding, some of the environment around it, so by default it doesn't spawn out in the open.

Going to be taking this advice in *partially* the opposite direction tbh. Thinking on it more I want only the south side to be exposed by default.

unit0016 committed:
:greg:

unit0016 committed:
Merge branch 'master' into tonight's-sosiest-loser

Erikafox reviewed (changes requested):
This is very detail-sparse for a crash map. Please include scrap and debris trailing the crash site.
![image](https://github.com/shiptest-ss13/Shiptest/assets/94164348/ae7e1d52-9896-4dbc-8311-2ba1e746d913)
subtype this so it has proper atmos please. Active turfs bad.
![image](https://github.com/shiptest-ss13/Shiptest/assets/94164348/39cd90fe-fc8f-4fbb-9992-ce98cabc0b8d)
ditto

Could use some creatures / definition stuff inside to show that it's been there a bit, and didn't experience the joys of atmospheric re-entry 20 minutes ago. Blown in snow piles, corpses, stuff like that.

unit0016 committed:
Merge branch 'master' into tonight's-sosiest-loser

unit0016 commented:
No updatepaths script, cringe

unit0016 committed:
Implements Dirt Icemoon Subtype, Further details the rear slightly

I miss when I made a ##path define for atmos comp but that shit isn't gonna be possible here thanks to future(tm) plans(tm). Next up: custom corpse spawners. the monkey's paw or something, you tell me.

unit0016 committed:
Call Me Jane Bethesda Because This Is Some Environmental Storytelling

Good god man, the tooling for testing ruins is super unfriendly. Map template place is really not the kind of tool I should be relying on. Also, did you know that /ruin/powered doesn't mean it SUPPORTS power, but means that it actually has MAGIC power? what the fuck?? Who thought this was okay and who HURT them..

unit0016 committed:
actually that's too MUCH wood

unit0016 committed:
getting slightly sloppy

unit0016 commented:
Did all of that. In the process, the following items have been added to the ruin and thus it's.. "loot" ""pool"":
- Two slags of corgi meat and a corgi hide
- A microwave
- A fueltank and mechanical toolbox
- a KNIFE (no!!)
- WOODDD. More wood! There's now enough for a baseball bat just lying around, but it's scattered.
- A double bedsheet.
- Brand spanking new mime and assistant husk corpse spawners. Also the generic corpse spawner icon has been redone to differentiate it a bit (purely because mob/landmarks.dmi had an icon state that didn't correspond to any job)

I was considering tacking on a pacman but decided against it last minute.

unit0016 committed:
Merge branch 'master' into tonight's-sosiest-loser

thgvr commented on code/modules/awaymissions/corpse.dm:
```diff
@@ -364,6 +367,14 @@
 	outfit = /datum/outfit/job/clown
 	icon_state = "corpseclown"
 
+/obj/effect/mob_spawn/human/mime
```
This isn't necessary - we don't support clown/mime content in ruins

unit0016 replied:
alrighty, I'll swap out the body for a ~~botanist~~ space bartender corpse (tacking on a plant analyzer is a nuh uh and I don't want to snowflake just for this) then, that's fitting enough.

unit0016 replied:
actually wait, does that encompass the ventriloquist dummy I already had in there considering that's part of the mime toolkit, or is that harmless?

thgvr replied:
Naw that's fine. Plant analyzers are in autolathes rn anyways

unit0016 committed:
delete now unused mime corpse, replace with (space) bartender corpse

awaiting feedback on if the dummy's good to keep

Erikafox commented:
> Did all of that. In the process, the following items have been added to the ruin and thus it's.. "loot" ""pool"":
> 
>     * Two slags of corgi meat and a corgi hide
> 
>     * A microwave
> 
>     * A fueltank and mechanical toolbox
> 
>     * a KNIFE (no!!)
> 
>     * WOODDD. More wood! There's now enough for a baseball bat just lying around, but it's scattered.
> 
>     * A double bedsheet.
> 
>     * Brand spanking new mime and assistant husk corpse spawners. Also the generic corpse spawner icon has been redone to differentiate it a bit (purely because mob/landmarks.dmi had an icon state that didn't correspond to any job)
> 
> 
> I was considering tacking on a pacman but decided against it last minute.

Go a little more wild with the debris and assorted random bullshit. Loot Pools are fake because most of this stuff is common enough anyways. Scatter around broken machinery bits so people can reassemble them, more wood, Just. More. It still feels weirdly empty to me.
I will duely inform you if it ends up being Too Much.

unit0016 committed:
Merge branch 'master' into tonight's-sosiest-loser

unit0016 committed:
i've lost track of what I've tacked on so I can't notarize it, but the SDMM view will be updated at least

unit0016 commented:
Hokay, that should be most of the whitespace filled up?

unit0016 commented:
Noticing MDB has an issue here where it shows the wrong icon state for chairs and chairs *specifically.* Weird!

unit0016 committed:
Merge branch 'master' into tonight's-sosiest-loser

Erikafox reviewed (approved):
Gonna see how this plays out today.

unit0016 committed:
Merge branch 'master' into tonight's-sosiest-loser

unit0016 committed:
Merge branch 'master' into tonight's-sosiest-loser

goober3 commented:
LGTM
